conftest.py

```diff
@@ -26,3 +27,27 @@ def pytest_collection_modifyitems(items):
         total_groups = int(os.environ.get("TOTAL_TEST_GROUPS", 1))
         group_num = int(md5(item.location[0]).hexdigest(), 16) % total_groups
         item.add_marker(getattr(pytest.mark, "group_%s" % group_num))
+
+
+@pytest.fixture(scope="session", autouse=True)
+def callattr_ahead_of_alltests(request):
+    """
+    Generate frontend assets before running any acceptance tests
+    """
+
+    # Do not build in CI because tests are run w/ `make test-acceptance` which builds assets
+    if os.environ["CI"] or os.environ["SKIP_ACCEPTANCE_UI_BUILD"]:
+        return
+
+    session = request.node
+    has_acceptance = False
+    for item in session.items:
+        if item.location[0].startswith("tests/acceptance/"):
+            has_acceptance = True
+            break
+
+    # Only run for acceptance tests
+    if not has_acceptance:
+        return
+
+    subprocess.call("NODE_ENV=development yarn webpack", shell=True)
```

Comment:
Can you change this to `.call(["yarn", "webpack"], env={"NODE_ENV": "development"})`? Just trying to avoid shell here.

Reply:
.

Reply:
Er,

```python
env={
    "NODE_ENV": "development",
    "PATH": os.environ["PATH"],
}
```